primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -238,20 +242,31 @@ where
 		// with substrate's generic `Vec<u8>` type. Basically this just means accepting that there
 		// will be a prefix of vector length (we don't need
 		// to use this).
```

Comment:
Maybe this comment can be updated/removed, since we're now using the encoded length for validation.